fetch_data.py

```diff
@@ -2,18 +2,21 @@
 from google.cloud import storage
 from dotenv import load_dotenv
 import os
+from pathlib import Path
 
 
 def configure():
     load_dotenv()
 
-creds = os.getenv("CREDS")
-# Set your GCS credentials
-storage_client = storage.Client.from_service_account_json(creds)
-bucket_name = 'rohan_bkk'
+# Create a storage_client using your GCS credentials
+gcs_key_path = Path(__file__).resolve().parent / 'service_account_key.json'
+gcs_key_path.write_text(os.getenv("KEY"))
```

Comment:
Ideally the `service_account_key.json` should be stored in an environment variable `KEY` or perhaps the even more self-documenting name `GCS_KEY`.